tutorials/cloud-spanner-aspnet/index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Using Cloud Spanner in ASP.NET
+description: Learn how to use Cloud Spanner with ASP.NET using C#.
+author: ben wulfe
+tags: Cloud Spanner, C#, SQL, ASP.NET, ADO.NET
+date_published: 2017-11-03
+---
+
+# Using Cloud Spanner in ASP.NET
+
+This tutorial describes how to use the [C# Google Cloud Spanner ADO.NET library](https://cloud.google.com/spanner/docs/getting-started/csharp/) to build a simple REST API for books and use the API to store and query books.
+
+[Cloud Spanner](https://cloud.google.com/spanner/) is a hosted SQL database, and the Cloud Spanner ADO.NET provider lets you to take advantage of Cloud Spanner in your C# applications. The Cloud Spanner ADO.NET library is fully asynchronous to the network level and is the recommended way to integrate Cloud Spanner APIs into your .NET applications.


```

Comment:
Oops--apologies. Extra "to" in this sentence. 
provider lets you to take advantage >> provider lets you take advantage

Reply:
fixed.  I'm assuming its all good to go now :)  Thanks again!